setup.py

```diff
@@ -0,0 +1,10 @@
+
+from setuptools import setup, find_packages
+
+setup(
+    name="change_state_traps",
+    version="0.1.0",
+    packages=["change_state_traps"],
+    author="Nepo Rojas",
+    python_requires='>=3.7'
```

Comment:
Crees que sea posible actualizar la versión de Python?
```suggestion
    python_requires='>=3.8'
```

Reply:
La última vez tuvimos problemas con la versión más nueva